release/release.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Thanks, this looks like a good idea, even though I am not sure if we need the `parse_all_gradle_properties` here.